ui/components/multichain/asset-picker-amount/index.scss

```diff
@@ -1,17 +1,87 @@
 .asset-picker-amount {
-  &__swap-button {
-    background: none;
+  // balance text elements
+  &__balance > *,
+  &__balance > div > * {
+    color: var(--color-text-alternative);
+    font-size: small;
+    padding: 0;
   }
 
+  // input for NFTs
+  &__input-nft {
+    &:focus-within {
+      border-color: inherit;
+    }
+
+    // input field itself
+    & div > div > input {
```

Comment:
Do we need this level of specificity?

Reply:
Removed this since it's redundant with the comment-labeled `input field` a bit below